examples/client-logger/app-insights-logger/src/components/App.tsx

```diff
@@ -0,0 +1,261 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { SharedCounter } from "@fluidframework/counter";
+import { ContainerSchema, IFluidContainer } from "@fluidframework/fluid-static";
+import { SharedMap } from "@fluidframework/map";
+import { SharedString } from "@fluidframework/sequence";
+
+import { CollaborativeTextArea, SharedStringHelper } from "@fluid-experimental/react-inputs";
+
+import { ContainerInfo, createFluidContainer, loadExistingFluidContainer } from "./ClientUtilities";
+
+/**
+ * Key in the app's `rootMap` under which the SharedString object is stored.
+ */
+const sharedTextKey = "shared-text";
+
+/**
+ * Key in the app's `rootMap` under which the SharedCounter object is stored.
+ */
+const sharedCounterKey = "shared-counter";
+
+/**
+ * Schema used by the app.
+ */
+const containerSchema: ContainerSchema = {
+	initialObjects: {
+		rootMap: SharedMap,
+	},
+	dynamicObjectTypes: [SharedCounter, SharedMap, SharedString],
+};
+
+/**
+ * Helper function to read the container ID from the URL location.
+ */
+function getContainerIdFromLocation(location: Location): string {
+	return location.hash.slice(1);
+}
+
+/**
+ * Populate the app's `rootMap` with the desired initial data for use with the client debug view.
+ */
+async function populateRootMap(container: IFluidContainer): Promise<void> {
+	const rootMap = container.initialObjects.rootMap as SharedMap;
+	if (rootMap === undefined) {
+		throw new Error('"rootMap" not found in initialObjects tree.');
+	}
+
+	// Set up SharedText for text form
+	const sharedText = await container.create(SharedString);
+	sharedText.insertText(0, "Enter text here.");
+	rootMap.set(sharedTextKey, sharedText.handle);
+
+	// Set up SharedCounter for counter widget
+	const sharedCounter = await container.create(SharedCounter);
+	rootMap.set(sharedCounterKey, sharedCounter.handle);
+	// Also set a couple of primitives for testing the debug view
+	rootMap.set("numeric-value", 42);
+	rootMap.set("string-value", "Hello world!");
+	rootMap.set("record-value", {
+		aNumber: 37,
+		aString: "Here is some text content.",
+		anObject: {
+			a: "a",
+			b: "b",
+		},
+	});
+}
+
+/**
+ * Root application component.
+ * Initializes the Fluid Container and displays app view once it is ready.
+ */
+export function App(): React.ReactElement {
+	// Load the collaborative SharedString object
+	// const { privateContainer, sharedContainer } = useContainerInfo();
+
+	const [sharedContainerInfo, setSharedContainerInfo] = React.useState<
```

Comment:
Nit: Since you only have a single container, you could probably just call this `container` and `setContainer`.